E2E-tests/config/substrate/devnet_nodes.json

```diff
@@ -157,6 +157,11 @@
                 "lock": 817414141,
                 "send": 298945143
             }
-        }
+        },
+        "permissioned_min_ariadne_d_param_value": 1,
+        "permissioned_max_ariadne_d_param_value": 5,
+        "trustless_min_ariadne_d_param_value": 1,
```

Comment:
Do we want to change the d param on devnet? If not we can leave it as is:
permissioned_min: 6
permissioned_max: 6
registered_min: 4
registered_max: 4